src/javascripts/modules/pillbar.js

```diff
@@ -0,0 +1,28 @@
+export default class Pillbar {
+  constructor(el) {
+    this.el = el
+    this.ticking = false
+    this.lastScrollY = 0
+
+    window.addEventListener('scroll', () => this.handleScroll())
+  }
+
+  handleScroll() {
+    this.lastScrollY = window.scrollY
+    if (!this.ticking) {
+      window.requestAnimationFrame(() => {
+        this.togglePillbar()
+      })
+    }
+    this.ticking = true
+  }
+
+  togglePillbar() {
+    this.ticking = false
+    if (this.lastScrollY > 420) {
```

Comment:
Since the title image is a variable height across breakpoints, your fixing position won't always be 420. You'll want to get the pillbar's position from the top of the viewport and save it in your constructor, then check against that variable here (instead of 420).

Reply:
You'd then also want to add a window resize handler that checks and resets that variable, since it'll be different based on how tall that title image is. Be sure to throttle it so that it doesn't continually run during resize. 200ms should be fine. You can use setTimeout and clearTimeout for this, or a utility method of some sort that you can import. Let me know if you need help with it.